main.py

```diff
@@ -119,9 +121,17 @@ def list_schedules():
     """
     schedules_list = []
     with client.context():
-        keys = SchedulesModel.query().fetch(keys_only=True)
-        for key in keys:
-            schedules_list.append(key.id())
+        verbose = request.args.get("verbose") == "true"
+        if (verbose):
```

Comment:
no need for parathesis around if-conditions in python

Reply:
fixed, thanks.